### DISCUSSION TITLE
Feature request: Archive all completed cards

### DISCUSSION THREAD

**Discussion by autonomygaps** [mgmeyers/obsidian-kanban#88] [Ideas & Feature Requests]
With the checkboxes now visible on the cards, it would be great if the there were a way to archive all "completed" cards, that is, to batch-move every card with a box checked to the archive in its list.

Ideally, it would be in the Command Palette , so that users could assign a hotkey to it.

**Comment by mgmeyers** [comment]
Thanks for the suggestion @autonomygaps, this has been implemented

**Reply by autonomygaps** [reply]
Fantastic!
For discoverability, it might be handy to add this option also to the "More Options" menu for each list.
